client/src/components/Ticketing/ticketingmanager/Season/SeasonInstancesPage.tsx

```diff
@@ -0,0 +1,134 @@
+import React, {useEffect, useState} from 'react';
+import {useNavigate} from 'react-router-dom';
+import {titleCase} from '../../../../utils/arrays';
+import {useFetchToken} from '../showings/ShowingUpdated/ShowingUtils';
+import {getSeasonImage, formatSeasonDate} from './seasonUtils';
+
+export interface Seasons {
+  seasonid: number;
+  name: string;
+  startdate: number;
+  enddate: number;
+  imageurl?: string;
+}
+
+/**
+ * Display all seasons page
+ *
+ * @module
+ * @returns Season Instances Page
+ */
+const SeasonInstancesPage = () => {
+  const navigate = useNavigate();
+  const [seasons, setAllSeasons] = useState<Seasons[]>([]);
+  const {token} = useFetchToken();
+
+  const getAllSeasons = async () => {
+    try {
+      const getAllSeasonsRes = await fetch(
+        process.env.REACT_APP_API_2_URL + '/season',
+        {
+          credentials: 'include',
+          method: 'GET',
+          headers: {
+            'Content-Type': 'application/json',
+            'Authorization': `Bearer ${token}`,
+          },
+        },
+      );
+
+      if (!getAllSeasonsRes.ok) {
+        throw new Error('Failed to fetch all seasons');
+      }
+
+      const seasonsData = await getAllSeasonsRes.json();
+      setAllSeasons(seasonsData);
+    } catch (error) {
+      console.error(error);
+    }
+  };
+
+  useEffect(() => {
+    getAllSeasons();
+  }, [token]);
+
+  return (
+    <div className='w-full h-screen overflow-x-hidden absolute'>
+      <div
+        className='md:ml-[18rem] md:mt-40 sm:mt-[11rem]
+       sm:ml-[5rem] sm:mr-[5rem] sm:mb-[11rem] h-full'
+      >
+        <section className={'flex flex-row justify-between w-full mb-14'}>
+          <h1
+            className='col-span-2 min-[678px]:col-span-1 font-bold text-5xl bg-clip-text
+           text-transparent bg-gradient-to-r from-sky-500
+            to-indigo-500'
+          >
+            Select Season
+          </h1>
+          <button
```

Comment:
The event version of this button also got refactored a bit

Reply:
Fixed!